### PR TITLE
Clarification/link to docs for WMS example

### PR DESCRIPTION
**Pull request by HeyStenson**
fixes https://github.com/mapbox/mapbox-gl-js-docs/issues/402

### CODE REVIEW

**Comment by andrewharvey** [comment]
+1, but it's also actually further up in the style spec doc at https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/ where it says

> By providing a URL to a WMS server that supports EPSG:3857 (or EPSG:900913) as a source of tiled data. The server URL should contain a "{bbox-epsg-3857}" replacement token to supply the bbox parameter.

File: docs/pages/example/wms.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
     map.on('load', function() {
         map.addSource('wms-test-source', {
             'type': 'raster',
+            // use the tiles option to specify a WMS tile source URL
+            // https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#raster-tiles
```
Comment (HeyStenson) [inline docs/pages/example/wms.html:14]:
```suggestion
            // https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/
```

**Comment by HeyStenson** [comment]
Thanks @andrewharvey! Updated with the right URL.

**Review by domlet — APPROVED** [review]
C'est bon!

Screen Shot 2020-09-23 at 5 30 59 PM

Screen Shot 2020-09-23 at 5 31 34 PM

**Pull request merged**